packages/web-components/fast-components-msft/src/styles/patterns/button.ts

```diff
@@ -51,29 +51,30 @@ export const BaseButtonStyles = css`
         opacity: var(--disabled-opacity);
     }
 
-    .start,
-    .end,
+    ::slotted([slot="start"]),
+    ::slotted([slot="end"]),
     ::slotted(svg) {
         ${
-            /* Glyph size and margin-left is temporary - 
+            /* Glyph size and margin-left is temporary -
             replace when adaptive typography is figured out */ ""
         } width: 16px;
+        outline: 1px solid #F0F;
```

Comment:
Can we remove this (assume it was for testing)?

Reply:
oops, my bad